src/Store.js

```diff
@@ -564,7 +564,14 @@ class Store {
     let records = []
 
     transaction(() => {
-      records = data.forEach(dataObject => this.createOrUpdateModel(dataObject))
+      records = data.forEach(dataObject => {
+        // Only build objects for which we have a type defined.
+        // And ignore silently anything else included in the JSON response.
+        // TODO: Put some console message in development mode
```

Comment:
Any reason not to do this right now? Seems easy enough

Reply:
I'm just worried about how to avoid doing it in production, how to tell if the code is being run in production, how to avoid posting the same message hundreds of times, etc, etc... and want to get this fix in as soon as possible